src/routes/(console)/project-[project]/auth/templates/+page.ts

```diff
@@ -1,9 +1,10 @@
 import { sdk } from '$lib/stores/sdk';
-import type { PageLoad } from './$types';
 
-export const load: PageLoad = async () => {
+export const load = async ({ parent }) => {
+    const { project } = await parent();
```

Comment:
this should be available from the project layout right?

Reply:
I think so

Reply:
this is still here 👀